recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -75,6 +92,20 @@ CleanupManager.addCleanupHandler(() => PreferenceExperiments.stopAllObservers())
 
 this.PreferenceExperiments = {
   /**
+   * Set the default preference value for active experiments that use the
+   * default preference branch.
+   */
+  async init() {
+    const store = await ensureStorage();
+    for (const experiment of Object.values(store.data)) {
+      const {expired, preferenceBranchType, preferenceName, preferenceValue} = experiment;
+      if (!expired && preferenceBranchType === "default") {
```

Comment:
I'm a bit surprised here - why do we not just set it for all of the pref experiments, including the ones using the user branch?